process/RequirementsForAdditionalFunctionality.md

```diff
@@ -0,0 +1,47 @@
+# Requirements for additional functionality to WebGPU
+
+This is a list of requirements which proposals for new functionality must adhere to in order to be standardized. New functionality is expected to usually take the form of a new [feature](https://gpuweb.github.io/gpuweb/#feature-index) to WebGPU.
+
+Adhering to these requirements is _necessary but not sufficient_ for proposals to be standardized by the WebGPU Community Group / Working Group. A proposal which meets these requirements may still be rejected by the CG / WG for other reasons.
+
+The list is expected to grow (or maybe even shrink!) over time. This list represents the CG's / WG's current processes. The initial version of this list was taken mostly from [this GitHub thread](https://github.com/gpuweb/gpuweb/issues/2310).
+
+## Glossary
+
+- "Browser engine" means "(Blink and Dawn) / (Gecko and wgpu (and maybe more?)) / (WebKit and WebGPU.framework) / etc."
+- "Desktop GPU" means "a non-mobile GPU."
+- "Device vendor" means "Intel / Nvidia / AMD / Apple / Qualcomm / etc."
+- ("Mobile GPU" is intentionally undefined, for now.)
+- "Native API" means "D3D / Vulkan / Metal." (The proprietary 3D graphics APIs used in game consoles would probably count as "native APIs," too, if there was a browser engine implementing WebGPU using those APIs.)
+- "Standardization" means all/any of:
+  - Putting proposals on the W3C recommendation track
+  - Committing to the WebGPU Git repository
+  - Adding anything in the WebGPU Github Team / Project / Wiki / Organization / etc.
+  - Mere discussion within the CG / WG official meetings.
+
+## Requirements
+
+In order for a proposal for additional functionality to be standardized in the WebGPU CG / WG,
+
+(These are numbered, so they can be referred to from other discussions.)
+1. A proposal for new functionality must be implementable on at least 2 different browser engines. Rationale: One browser engine should not be able to unilaterally add functionality to the web platform.
```

Comment:
so, just to clarify, Edge + Chrome wouldn't count as 2 engines, right?

Reply:
That's correct, since the relevant code isn't distinct between those two. I can add more clarifications here.